src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -26,26 +29,60 @@ public class PostHearingReviewAboutToSubmitHandler implements PreSubmitCallbackH
     public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
         requireNonNull(callback, "callback must not be null");
         requireNonNull(callbackType, "callbacktype must not be null");
-
         return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
             && callback.getEvent() == EventType.POST_HEARING_REVIEW
             && isPostHearingsEnabled;
     }
 
     @Override
     public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
-                                                          String userAuthorisation) {
+        String userAuthorisation) {
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
 
+        updateCaseStatus(caseData);
         PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(caseData);
 
-        String caseId = caseData.getCcdCaseId();
+        excludePanelMembers(caseData, response);
 
+        String caseId = caseData.getCcdCaseId();
         PostHearingReviewType typeSelected = caseData.getPostHearing().getReviewType();
         log.info("Review Post Hearing App: handling action {} for case {}", typeSelected,  caseId);
 
         SscsUtil.clearDocumentTransientFields(caseData);
 
         return response;
     }
+
+    protected void updateCaseStatus(SscsCaseData caseData) {
+        PostHearing postHearing = caseData.getPostHearing();
+        if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
+            caseData.setState(State.NOT_LISTABLE);
+            caseData.setInterlocReviewState(InterlocReviewState.AWAITING_ADMIN_ACTION);
+        }
+    }
+
+    protected void excludePanelMembers(SscsCaseData caseData, PreSubmitCallbackResponse<SscsCaseData> response) {
+        PostHearing postHearing = response.getData().getPostHearing();
+        if (PostHearingReviewType.SET_ASIDE.equals(postHearing.getReviewType())
+            && SetAsideActions.GRANT.equals(postHearing.getSetAside().getAction())
+        ) {
+            log.info("Set Aside is granted. Excluding panel members for case {}", caseData.getCcdCaseId());
+
+            HearingDetails latestHearing = caseData.getLatestHearing().getValue();
+            List<String> panelMemberIds = latestHearing.getPanelMemberIds(); // TODO test this list to confirm working as expected
+            String judgeId = latestHearing.getJudgeId();
+            ArrayList<JudicialUserBase> panelMembersToExclude = panelMemberIds.stream()
```

Comment:
what are we doing if we don't have the IDs? Are we showing the warning to tell the user to manually exclude? Are we gunna try and find the panel member IDs?

Reply:
On discussion with BA, the business will manually update all existing cases prior to post hearing going live. This will mean that every relevant case will have IDs available.

Reply:
agreed